### ISSUE TITLE
Game crash

### ISSUE THREAD

**Issue by Pyrox300** [brandon3055/Draconic-Evolution#408]
My game crashes in creativ when ever i try to craft the Draconic Armor or the Whyvern armor

**Comment by cylek56** [comment]
They aren't implemented yet ;)

**Comment by Morpheus1101** [comment]
The Draconic & Wyvern armor are indeed implamented and craftable and i have used them, however the Draconic Armor needs to have the Wyvern Armor upgraded through Infusion to get Draconic

**Comment by covers1624** [comment]
Yes, a crash report is vital, just saying it crashes, is not enough information. We test everything and i have ideas as to what this could be, but i need your crash report to actually do anything about it.
